app/controllers/red_line_boundary_change_validation_requests_controller.rb

```diff
@@ -22,14 +21,29 @@ def create
 
     if @red_line_boundary_change_validation_request.save
       redirect_to(
-        (session.delete(:return_to) || create_request_redirect_url),
+        create_request_redirect_url,
         notice: t(".validation_request_for")
       )
     else
       render :new
     end
   end
 
+  def create_request_redirect_url
+    case return_to
+    when nil
+      super
+    when planning_application_sitemap_url(@planning_application)
```

Comment:
The default is to return to the previous page. However there is one specific user journey, where the user is directed to the request form from another form, where we want need to overwrite this behavious.